starter-code/src/main.js

```diff
@@ -42,9 +44,56 @@ $(document).ready(function(){
   // Add all the div's to the HTML
   document.getElementById('memory_board').innerHTML = html;
   // Bind the click event of each element to a function
-$('.back').on('click', function () {
-   
-});
+
+  $('.card').on('click', function (event) {
+
+
+    event.stopPropagation();
+
+    var thisCard = $(this).children();
+
+    if ($(thisCard[0]).hasClass("back")) {
+
+      $(thisCard[0]).removeClass("back").addClass("front front-active"); //Creating front-active and back-active for comparing in pairs and not turning all cards if you're wrong.
+      $(thisCard[1]).removeClass("front").addClass("back back-active");
+
+      currentCardsAttribute.push(event.currentTarget.id); // Pushing both attributes into array for comparing
+
+
+      if (currentCardsAttribute.length > 0 && currentCardsAttribute.length % 2 === 0) {
+        thereAreEquals(currentCardsAttribute);
+      }
+
+    } else {
+
+      $(thisCard[0]).removeClass("front front-active").addClass("back");
+      $(thisCard[1]).removeClass("back back-active").addClass("front");
+
+    }
+  });
+
 });
 
 
+function thereAreEquals(array) {
+
+
+  if (array[0] === array[1]) {
+
+    memoryGame.pickedCards.push(array); // Pushing the correct cards into the object created in memory.js
+
+    $('.front-active').removeClass('front-active');  //Removing active classes for not turning the correct cards.
+    $('.back-active').removeClass('back-active');
+
+    currentCardsAttribute = [];
+  }
+
+  else {
+    $('.front-active').removeClass('front front-active').addClass('back');
+    $('.back-active').removeClass('back back-active').addClass('front');
+
+    currentCardsAttribute = [];
+
+  }
```

Comment:
This is a good example of code that should be on the Memory Game constructor. The checkIfPair function would be something like this:
```
MemoryGame.prototype.checkIfPair = function (firstCard, secondCard) {
  this.pairsClicked++;
  if (firstCard === secondCard) {
    memoryGame.pairsGuessed++;
  }
  return firstCard === secondCard;
}
```